index.html

```diff
@@ -40,12 +40,14 @@
 	<div class="wrapper">
 		<header>
 			<h1><span>Tech</span>Week</h1>
-			<p>Dublin City University 9<sup>th</sup> - 13<sup>th</sup> March 2015</p>
+			<p>Dublin City University 9<sup>th</sup> - 13<sup>th</sup> March 2016</p>
+			<h4>We are currently organizing the sechedual for TechWeek</h4>
+
 		</header>
 
 		<!-- Countdown is show up to 10 minutes before the event -->
 		<div id="countdown">
-			TechWeek Starts in: 
```

Comment:
Atom is very strict about about whitespaces at the end of a file. It always parses out that stuff. 
that is why there is a few lines of diff that do nothing really